spec/features/person_maintenance_spec.rb

```diff
@@ -398,6 +417,21 @@
       end
     end
 
+    context "with an external user", user: :readonly do
```

Comment:
I think this needs updating.
```suggestion
    context "with an external user", user: :external_user do
```